### PR TITLE
PMM-8412 update to Grafana 8

### PR DESCRIPTION
**Pull request by fabio-silva**
https://jira.percona.com/browse/PMM-8412

### CODE REVIEW

File: src/components/CopyToClipboard/CopyToClipboard.styles.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { GrafanaTheme } from '@grafana/data';
-import { css } from 'emotion';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
 
-export const getStyles = (theme: GrafanaTheme) => {
-  const { spacing, colors, border, zIndex } = theme;
+export const getStyles = ({ v1}: GrafanaTheme2) => {
```
Comment (tiagomotasantos) [inline src/components/CopyToClipboard/CopyToClipboard.styles.ts:4]:
```suggestion
export const getStyles = ({ v1 }: GrafanaTheme2) => {
```

File: src/setupTests.ts [hunk 2]
```diff
@@ -8,3 +8,17 @@ import 'jest-canvas-mock';
...
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  })),
+});
```
Comment (tiagomotasantos) [inline src/setupTests.ts:24]: Missing EOF line

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: src/components/CopyToClipboard/CopyToClipboard.styles.ts (hunk 1 above)
Comment (fabio-silva) [reply src/components/CopyToClipboard/CopyToClipboard.styles.ts]: 👍

File: src/setupTests.ts (hunk 2 above)
Comment (fabio-silva) [reply src/setupTests.ts]: 👍

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**